benchmarkers/benchmarker.py

```diff
@@ -0,0 +1,61 @@
+import time
+import re
+from hpobench.benchmarks.ml.xgboost_benchmark import XGBoostBenchmark
+from hpobench.benchmarks.ml.svm_benchmark import SVMBenchmark
+from loguru import logger
+
+
+
+class Benchmarker:
+    def __init__(self, task_id, model_name="xgb"):
+        self.task_id = task_id
+        self.config_space = None
+        if model_name == "xgb":
+            self.model_name = "XGBoost"
+            self.benchmarker = XGBoostBenchmark(task_id=task_id)
+            self.fidelity = {"n_estimators": 2000, "subsample": 1}
+        elif model_name == "svm":
+            self.model_name = "SupportVectorMachine"
+            self.benchmarker = SVMBenchmark(task_id=task_id)
+            self.fidelity = {"subsample": 1}
+        else:
+            raise Exception("Need a model name: xgb or svm")
+
```

Comment:
we dont need lines 24 - 48
we just need to `return dict((i.name, (i.lower, i.upper)) for i in self.benchmarker.get_configuration_space(seed).get_hyperparameters())`